src/app/Traits/HistoryTracker.php

```diff
@@ -10,13 +10,9 @@ trait HistoryTracker
 
     protected static function bootHistoryTracker()
     {
-        self::created(function ($model) {
-            $model->saveHistory();
-        });
+        self::created(fn($model) => $model->saveHistory());
 
-        self::updated(function ($model) {
-            $model->saveHistory();
-        });
+        self::updated(fn($model) => $model->saveHistory());
```

Comment:
@checkerschaf we use it here.

Reply:
@raftx24 I know that you use it NOW but not YET. In my opinion it's not a big improvement to use it at the moment but I don't mind.